platform/openharmony/os_openharmony.cpp

```diff
@@ -0,0 +1,462 @@
+/**************************************************************************/
+/*  os_openharmory.cpp                                                    */
```

Comment:
```suggestion
/*  os_openharmony.cpp                                                    */
```
This should have been caught by CI, will investigate